specification/resources/resource-manager/Microsoft.Resources/stable/2019-05-10/resources.json

```diff
@@ -1096,6 +1136,42 @@
         }
       }
     },
+    "/providers/{resourceProviderNamespace}": {
+      "get": {
+        "tags": [
+          "Providers"
+        ],
+        "operationId": "Providers_GetAtTenant",
```

Comment:
>Providers_GetAtTenant [](start = 24, length = 21)

Let's do `Providers_GetAtTenantScope`

Reply:
Done, and changed Providers_ListAll to Providers_ListAtTenantScope.